tests/testVue.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
-i= basename "$PWD"
+mkcdir ()
```

Comment:
Please move this file to the `test` folder rather than the `tests` folder

Reply:
Part of 6dee007